src/DICOMWeb/index.test.js

```diff
@@ -0,0 +1,54 @@
+import * as DICOMWeb from './index.js';
+
+describe('Top level exports', () => {
+  test('should export the modules getAttribute, getAuthorizationHeader, getModalities, getName, getNumber, getString', () => {
+    const expectedExports = [
+      'getAttribute',
+      'getAuthorizationHeader',
+      'getModalities',
+      'getName',
+      'getNumber',
+      'getString',
+    ].sort();
+
+    const exports = Object.keys(DICOMWeb.default).sort();
+
+    expect(exports).toEqual(expectedExports);
+  });
+
+  test('should be able to call getAttribute function', () => {
+    DICOMWeb.default.getAttribute = jest.fn();
+    DICOMWeb.default.getAttribute();
+    expect(DICOMWeb.default.getAttribute).toBeCalled();
+  });
```

Comment:
I think we only need the expected exports test for this one. I'm not sure we're actually testing anything with each of these other tests. When we set the property equal to `jest.fin()`, if it wasn't there before, it now is, so of course we can call it. And we already know it's there because of the first test.

Reply:
done. Thanks for the feedback!